application.go

```diff
@@ -99,6 +98,8 @@ type Application struct {
 	LastTaskFailure       *LastTaskFailure        `json:"lastTaskFailure,omitempty"`
 	Fetch                 *[]Fetch                `json:"fetch,omitempty"`
 	IPAddressPerTask      *IPAddressPerTask       `json:"ipAddress,omitempty"`
+	Env                   map[string]string       `json:"env,omitempty"`
+	Secrets               map[string]Secret       `json:"secrets,omitempty"`
```

Comment:
Also wondering here if we actually need a pointer to a map in order to facilitate serializing empty (i.e., cleared) secrets.